packages/storybook/stories/data-table.mdx

```diff
@@ -0,0 +1,54 @@
+import { Table } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
+import * as TableStories  from './data-table.stories';
+
+<Meta of={TableStories} />
+
+# Data Table
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+The data table is a structural component used to efficiently display and organize data [1].
```

Comment:
comme nous avons changé la façon de regrouper les composants, j'enlèverais "is a structural component"